File: 1er_parcial_recuperatorio/java/PrimerParcialRecuperatorio/src/modelos/Paquete.java
package modelos;

public class Paquete {

    private double peso;
    private double volumen;

    public Paquete(double peso, double volumen) {
        this.peso = peso;
        this.volumen = volumen;
    }

    public double getPeso() {
        return peso;
    }

    public double calcularPesoVolumetrico() {
        return volumen / 5000;
    }

    @Override
    public String toString() {
        return "Peso: " + peso + "\n" +
               "Volumen: " + volumen + "\n" +
               "Peso volumétrico: " + calcularPesoVolumetrico();
    }
}

File: 1er_parcial_recuperatorio/python/modelos/Paquete.py
class Paquete:

    def __init__(self, peso, volumen):
        self.__peso = peso
        self.__volumen = volumen

    @property
    def peso(self):
        return self.__peso

    def calcular_peso_volumetrico(self):
        return self.__volumen / 5000

    def __str__(self):
        return (
            f"Peso: {self.__peso}\n"
            f"Volumen: {self.__volumen}\n"
            f"Peso volumétrico: {self.calcular_peso_volumetrico()}"
        )
